test/test-list.h

```diff
@@ -275,6 +275,9 @@ TEST_DECLARE   (fs_fstat)
 TEST_DECLARE   (fs_access)
 TEST_DECLARE   (fs_chmod)
 TEST_DECLARE   (fs_copyfile)
+#if !defined(_WIN32) && !defined(_AIX) && !defined(__APPLE__)
```

Comment:
Suggestion: define unconditionally and move the `#if` guard inside the function in test/test-fs-copyfile.c, doing a RETURN_SKIP on unsupported platforms?